src/test/java/com/easypost/ThreadSafeTest.java

```diff
@@ -0,0 +1,55 @@
+package com.easypost;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Address;
+import com.easypost.model.AddressThreadSafe;
+import com.easypost.service.EasyPostClient;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeAll;
+
+public class ThreadSafeTest {
+    private static TestUtils.VCR vcr;
+
+    @BeforeAll
+    public static void setup() throws EasyPostException {
+        vcr = new TestUtils.VCR("address_thread_safe", TestUtils.ApiKey.TEST);
+    }
+
+    // Test creating/retrieving/verifying an address (thread-safe) with EasyPostClient object
+    @Test
+    public void testAddress() throws EasyPostException {
+        vcr.setUpTest("create");
+        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_TEST_API_KEY"));
+        
+        AddressThreadSafe address = client.Address().create(Fixtures.caAddress1());
```

Comment:
Thought: I don't think we'd want to have user instantiating each namespace, we should be doing this behind the scenes for them when it's called right? Seems like a lot of overhead that could easily be forgotten.

Reply:
Not sure if I follow. Could you provide an example?